content/faq.md

```diff
@@ -0,0 +1,103 @@
+### What really is Cryptocurrency
```

Comment:
I would not include this question on FAQ. It's impossible to give a short comprehensive answer to what is a cryptocurrency. The answer to "What is Monero" is sufficient, in my opinion as it outlines the most important features of Monero a new user should know. The rest is just technicalities.